### ISSUE TITLE
This plugin today not working any more (Firefox 61.0.2)

### ISSUE THREAD

**Issue by ea1234567** [bijij/ViewImage#155]
This plugin today not working any more (Firefox 61.0.2)

![image](https://user-images.githubusercontent.com/40317198/44144424-af3e2ffc-a09c-11e8-8e60-e27b40a44442.png)

**Comment by thisisjvgrace** [comment]
yeah, not working for me too, on Google Chrome Version 68.0.3440.106 (Official Build) (64-bit)
